doc/tutorial/creating-autoinstall-configuration.rst

```diff
@@ -0,0 +1,96 @@
+Creating autoinstall configuration
+===================================
+
+When any system is installed using the Ubuntu installer, an autoinstall file
+for repeating the installation is created at
+:code:`/var/log/installer/autoinstall-user-data`.
+
+
+The structure of an autoinstall configuration
+---------------------------------------------
+
+See the :ref:`ai` for full details on the supported autoinstall directives.
+
+A minimal autoinstall configuration in the
+:external+cloud-init:ref:`user_data_formats-cloud_config` format looks like:
```

Comment:
I think this section would be better served by linking to the `providing-autoinstall` page instead of the cloud-init page.